Lib/test/test_tkinter/test_misc.py

```diff
@@ -491,6 +491,8 @@ def test2(e): pass
         f.unbind(event, funcid)
         script = f.bind(event)
         self.assertNotIn(funcid, script)
+        self.assertIn(funcid2, script)
+        self.assertEqual(f.bind(), (event,))
```

Comment:
To properly test the unbind code (see comment), I think this should bind 3 scripts and unbind the middle one.

Reply:
I would also like to test that the other two funcids still work in unbind.

Reply:
There is a separate test, `test_bind_events()`, that tests that registered functions are called when the event occurs. Although it does not cover unbinding.

It is not so easy to make it working. For example, currently I cannot make it working in this test, an I do not know why. Maybe it does not work with keyboard events.

Reply:
Well, I found how to do this. Will see whether it works on Windows.

Reply:
The expanded test, not visible above, is what I proposed.  When I just ran test_misc, whether by itself or with test_tkinter, whether directly or with regrtest, test_bind_events hangs with two blank tk windows on the screen.  Hence the CI test timed out.  When I close the first of the two windows, tests 'continue' after this error:
```
ERROR: test_bind_events (__main__.BindTest.test_bind_events)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "F:\dev\3x\Lib\test\test_tkinter\test_misc.py", line 709, in test_bind_events
    root.wait_visibility()  # needed on Windows
    ^^^^^^^^^^^^^^^^^^^^^^
  File "F:\dev\3x\Lib\tkinter\__init__.py", line 752, in wait_visibility
    self.tk.call('tkwait', 'visibility', window._w)
_tkinter.TclError: window "." was deleted before its visibility changed
```
This is due to the added "self.frame.wait_visibility()" in BindTest.setUp.  With root deleted to let tests continue, the remaining BindTest functions fail with various messages.

`DefaultRootTest.test_default_root` also fails, with 
"AttributeError: module 'tkinter' has no attribute '_default_root'.".  A bad inter-testcase dependency?  The following `DefaultRootTest.test_getboolean` and everything after passes.

With the the test_bind_events wait_visibility commented out, there is this error.
```
FAIL: test_bind_events (__main__.BindTest.test_bind_events)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "F:\dev\3x\Lib\test\test_tkinter\test_misc.py", line 729, in test_bind_events
    self.assertEqual(events, [
AssertionError: Lists differ: [] != [('frame', <tkinter.Frame object .!topleve[172 chars]me>)]
```

With the setUp wait_visibility instead commented out, there is this error:
```
>FAIL: test_unbind2 (__main__.BindTest.test_unbind2)
Traceback (most recent call last):
  File "F:\dev\3x\Lib\test\test_tkinter\test_misc.py", line 496, in test_unbind2
    self.assertEqual(events, ['a', 'b', 'c'])
AssertionError: Lists differ: [] != ['a', 'b', 'c']
```
With the setUp call instead moved into test_unbind2 either at the top or as `f.wait_visibility()` before the generate events call, there is no error.